src/test/java/no/unit/nva/doi/transformer/utils/DataciteRelatedIdentifierTypeTest.java

```diff
@@ -0,0 +1,49 @@
+package no.unit.nva.doi.transformer.utils;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+class DataciteRelatedIdentifierTypeTest {
+
+    @DisplayName("DataciteRelatedIdentifiers exist")
+    @ParameterizedTest
+    @ValueSource(strings = {"ARK", "ARXIV", "BIBCODE", "DOI", "EAN13", "EISSN", "HANDLE", "IGSN", "ISBN", "ISSN",
+            "ISTC", "LISSN", "LSID", "PMID", "PURL", "UPC", "URL", "URN", "W3ID", "UNKNOWN_IDENTIFIER"})
+    void dataciteRelatedIdentifiersExist(String value) {
+        assertNotNull(DataciteRelatedIdentifierType.getByCode(value));
+    }
+
+    @DisplayName("DataciteRelatedIdentifiers can be retrieved by code")
+    @ParameterizedTest
+    @ValueSource(strings = {"ARK", "ARXIV", "BIBCODE", "DOI", "EAN13", "EISSN", "HANDLE", "IGSN", "ISBN", "ISSN",
+            "ISTC", "LISSN", "LSID", "PMID", "PURL", "UPC", "URL", "URN", "W3ID"})
+    void dataciteRelatedIdentifiersGetByCodeReturnCodeWhenInputCodeIsValid(String value) {
+        assertEquals(value, DataciteRelatedIdentifierType.getByCode(value).getCode().toUpperCase());
+    }
+
+    @DisplayName("DataciteRelatedIdentifiers have description")
+    @ParameterizedTest
+    @ValueSource(strings = {"ARK", "ARXIV", "BIBCODE", "DOI", "EAN13", "EISSN", "HANDLE", "IGSN", "ISBN", "ISSN",
+            "ISTC", "LISSN", "LSID", "PMID", "PURL", "UPC", "URL", "URN", "W3ID"})
+    void dataciteRelatedIdentifiersGetDescriptionReturnCodeWhenInputCodeIsValid(String value) {
```

Comment:
rename to `dataciteRelatedIdentifiersGetDescriptionReturnsDescriptionWhenInputCodeIsValid`